pep/node/server.js

```diff
@@ -13,6 +13,21 @@ module.exports = function(emberDistPath) {
         res.sendFile(`${emberDistPath}/assets/images/favicons/favicon.ico`);
     });
 
+    server.all('/toc.php', (req, res) => {
+        const journal = req.query.journal;
+        const volume = req.query.volume;
+        if (volume) {
+            res.redirect(301, `/browse/${journal}/volumes/${volume}`);
+        } else {
+            res.redirect(301, `/browse/${journal}/volumes`);
+        }
+    });
+
+    server.all('/document.php', (req, res) => {
+        const id = req.query.id;
+        res.redirect(301, `/browse/document/${id}`);
+    });
```

Comment:
won't hold this up as I'm sure these changes are a priority, but @adistasio @lhorne-gavant I wonder if redirects like this would be better off in the api gateway/lambda layer, if we can config any sort of mod_rewrite-like rules easily there? 

Just so these old requests dont need to get to and spin up the fastboot lambda, only to be redirected to another fastboot lambda request.